rust/src/push/base_rules.rs

```diff
@@ -241,6 +225,21 @@ pub const BASE_APPEND_OVERRIDE_RULES: &[PushRule] = &[
         default: true,
         default_enabled: true,
     },
+    // We don't want to notify on edits *unless* the edit directly mentions a
+    // user, which is handled above.
+    PushRule {
+        rule_id: Cow::Borrowed("global/override/.org.matrix.msc3958.suppress_edits"),
+        priority_class: 5,
+        conditions: Cow::Borrowed(&[Condition::Known(KnownCondition::EventPropertyIs(
+            EventPropertyIsCondition {
+                key: Cow::Borrowed("content.m\\.relates_to.rel_type"),
```

Comment:
```suggestion
                key: Cow::Borrowed(r"content.m\.relates_to.rel_type"),
```

Might be an easier read? (Those are called raw strings)

Reply:
That would be fine too, yeah. I didn't realize Rust had those like Python does. 👍